backend.py

```diff
@@ -95,6 +104,13 @@ def confidence(self):
     def noise(self):
         return self._data.noise.reshape(self.height, self.width)
 
+    @property
+    def ir_img(self):
+        if self._ir_img is None:
+            data = self._ir_data.data.reshape(self.height, self.width)
+            self._ir_img = cv2.cvtColor(data, cv2.COLOR_GRAY2BGR)
```

Comment:
What is the reason to convert this image to BGR? Pupil should be able to support single channel gray images.

Reply:
Turns out, some of the calibration methods, e.g. Fingertip Calibration, expect a BGR image.